### PR TITLE
Add support for EvaluateLowSampleCountPercentile field on percentile …

### PR DESCRIPTION
**Pull request by jvandervelden**

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -161,6 +161,7 @@ class AlertsPlugin {
         alarm.Properties.Statistic = definition.statistic
       } else {
         alarm.Properties.ExtendedStatistic = definition.statistic
+        alarm.Properties.EvaluateLowSampleCountPercentile = definition.evaluateLowSampleCountPercentile
```
Comment (AntonBazhal) [inline src/index.js:164]: I'm not sure whether CFN accepts `undefined`, but we can test

**Review by AntonBazhal — APPROVED** [review]
version bump?

File: src/index.js (hunk 1 above)
Comment (jvandervelden) [reply src/index.js:164]: I tested and it defaults to 'evaluate'. When you json stringify an undefined value it removes it: 
```
> JSON.stringify({ a: undefined, b: 'asdf' });
'{"b":"asdf"}'
```

**Comment by jvandervelden** [comment]
> version bump?

The original repo does separate commit version bumps. So I'd rather follow that for this merge, so it doesn't mess with that PR.

**Pull request merged**